meerk40t/core/elements/shapes.py

```diff
@@ -234,17 +234,50 @@ def element_line(command, x0, y0, x1, y1, data=None, post=None, **kwargs):
         post.append(classify_new(data))
         return "elements", data
 
+
```

Comment:
I'm not sure shapes is entirely the right place for these commands. Not sure a better place but we might need to sort these properly at some point.